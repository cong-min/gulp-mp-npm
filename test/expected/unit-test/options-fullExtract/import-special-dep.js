import './dist/weui-miniprogram/index.js';
import './dist/weui-miniprogram/index.js';
import './dist/weui-miniprogram/loading/loading.js';
