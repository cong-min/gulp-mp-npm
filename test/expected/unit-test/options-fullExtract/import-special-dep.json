{
  "usingComponents": {
    "mp-msg": "./dist/weui-miniprogram/msg/msg",
    "mp-sticky": "./dist/@miniprogram-component-plus/sticky",
    "page": "./local-dep/page/page"
  }
}
